.github/workflows/test.yml

```diff
@@ -197,3 +197,22 @@ jobs:
           targets: binary-smoketest
           set: |
             *.platform=${{ matrix.platform }}
+
+  vm:
+    needs:
+      - validate-dco
+    uses: ./.github/workflows/.vm.yml
+    strategy:
+      fail-fast: false
+      matrix:
+        template:
+          # EL 8 is used for running the tests with cgroup v1.
+          # Do not upgrade this to EL 9 until formally deprecating the cgroup v1 support.
+          #
+          # FIXME: use almalinux-8, then probably no need to keep oraclelinux-8 here.
+          # On almalinux-8, port forwarding tests are failing:
+          # https://github.com/moby/moby/pull/49819#issuecomment-2815676000
+          - template://oraclelinux-8 # Oracle's kernel 5.15
+          # - template://almalinux-8 # kernel 4.18
```

Comment:
Let me just use oraclelinux-8 (kernel 5.15) and call it a day.

The failures on almalinux-8 (kernel 4.18) have to be analyzed though.